### DISCUSSION TITLE
[Suggestion] omit markdown symbol for code

### DISCUSSION THREAD

**Discussion by erikmagic** [linpengcheng/PurefunctionPipelineDataflow#1] [Ideas]
Hey,

Sounds like very nice work. If I may I would like to suggest to remove the back tick with the clojure before codes.
If your parser could automatically detect clojure code and format it into markdown, that would be much cleaner.
I see the potential for heavily documented source code to work with your parser in order to generate detailed code documentation.

Anyway, keep it up it looks good!
EO

**Comment by linpengcheng** [comment]
In the latest version, it is implemented using Babaska, which defaults to clojure source files, and there is no need to mark Clojure code blocks. It's just that the article hasn't been revised yet.
Code sample: https://github.com/linpengcheng/ClojureBoxNpp/blob/master/Notepad%2B%2B/tools/MarkdownLiteraryProgramming/MLP_Clojure_Example.clj
